order-management-service/app/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencyManagement {
 }
 
 dependencies {
+    compile 'org.springframework.cloud:spring-cloud-starter-eureka:1.4.7.RELEASE'
```

Comment:
can be get rid of version here?

Reply:
I can move it into properties file not BOM (i won't work with this crap)